config/database.yml

```diff
@@ -16,12 +22,13 @@ test:
 
 staging:
   <<: *defaults
-  collation: 'es_ES.UTF-8'
-  ctype: 'es_ES.UTF-8'
+  # collation: 'es_ES.UTF-8'
+  # ctype: 'es_ES.UTF-8'
```

Comment:
maybe we can ✂️ this commented lines? I think that was only necessary for our current and old PostgreSQL setup

Reply:
yep!